proxy/src/project.rs

```diff
@@ -50,21 +52,26 @@ pub struct Project {
     pub registration: Option<Registration>,
     /// High-level statistics about the project
     pub stats: coco::Stats,
+    /// The user to which all source browsing queries are scoped to.
+    pub default_user: identity::Identity,
```

Comment:
Does this belong here? I would think it's part of the Metadata (since that's where we get it from)?

Reply:
The metadata of a project is the bucket for user defined input during project creation and update. It's not derived or generated data.

Reply:
Ah I see, so it's not the project identity.

Reply:
Had a chat with @xla IRL, we decided that the ordering of the `GET /source/revisions` response (which also contains peers) will determine the default_peer. It's going to be the first in the list.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/620/commits/06e9bdacf9c31c800b28cf6779cb1559d4323175